.github/workflows/pr.yml

```diff
@@ -0,0 +1,19 @@
+name: PR
+
+on: pull_request
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Setup Chart Testing
+        uses: helm/chart-testing-action@v2.0.1
+
+      - name: Run chart-testing (lint)
+        run: ct lint --all
```

Comment:
should not we lint only changed charts?

Reply:
I tend to prefer linting all charts regardless as it is not a lengthy process. For integration testing if we started to enable that aspect, would only target changed charts

Reply:
Hm, but updating `chart_schema.yaml` might make linting fails on charts that are not a part of PR. Would that be ok?

Reply:
Thats a good point. @deweya @sbose78 any thoughts?

Reply:
Yeah it's something to consider. I use just `ct lint` since only changed charts will be linted. You don't really need to lint unchanged charts but I get doing it to be safe.

I think it's best to do just `ct lint` for the point @pedjak makes. I can see a quick change turning into a large PR if we need to update chart_schema.yaml in the future. If we do make a change to chart_schema.yaml, chart maintainers will just need to update their Chart.yaml file in their next update.

Reply:
Updated